src/integrations/typedtables.jl

```diff
@@ -80,7 +80,7 @@ end
     push_exprs = Expr(:block)
     for i in 1:n
         if tt.parameters[2].parameters[i] <: NullableArray
-            ex = :( push!(tt.data[$i], Nullable(row[$i]) ))
+            ex = :( push!(tt.data[$i], isnull(row[$i]) ? Nullable() : Nullable(row[$i]) ))
```

Comment:
Should use `Nullable{typeof(row[$i])}()` or something equivalent to ensure type stability.